lambdas/services/pds_patient.json

```diff
@@ -0,0 +1,411 @@
+{
```

Comment:
Since this is a mock / expected result for the service at this point in time, should we keep these in a separate folder? e.g. services/mocks/pds_patient.json?